drivers/net/ethernet/msm/rndis_ipa.c

```diff
@@ -29,7 +29,7 @@
 #include <linux/workqueue.h>
 
 #define CREATE_TRACE_POINTS
-#include "rndis_ipa_trace.h"
+#include <trace/rndis_ipa_trace.h>
```

Comment:
I understand the need/want for this type of change, but the way kernels are built in an android system means this isn't needed.  So I don't think this is really necessary.

But it would be nice :)

Same with all of the other include changes in this patch.